src/Packages/Passport/Runtime/Scripts/Public/Passport.cs

```diff
@@ -44,13 +44,30 @@ private Passport()
 #endif
         }
 
-        public static UniTask<Passport> Init(string clientId, string environment, string? redirectUri = null)
+        /// <summary>
+        ///     Initialises Passport
+        /// </summary>
+        /// <param name="clientId">The client ID</param>
+        /// <param name="environment">The envronment to connect to</param>

```

Comment:
typo in environment. Extra spaces in `Initialises Passport` above, not sure if intentional

Reply:
The space I added in front of "Initialises Passport" is intentional after checking out other SDK docs, but since it doesn't match our current style, I'll remove it.

Fixing typo, thanks!